tests/test_jenkins.py

```diff
@@ -161,6 +162,24 @@ def test_fetch_empty(self):
 
         self.assertEqual(len(builds), 0)
 
+    @httpretty.activate
```

Comment:
But we avoid to get the job, not builds. Checking if we can do something like that because it is a nice improvement ... there is no job field in the builds, so we can not do this checking.